### ISSUE TITLE
Error Cannot read properties of null (reading 'getPassword')

### ISSUE THREAD

**Issue by LauSere** [formulahendry/vscode-mysql#208]
#Type: <b>#195308</b>

Cuando voy a la pestaña mysql sale ese error: Cannot read properties of null (reading 'getPassword'). Única extensión instalada es MySQL v0.4.1 de Jun Han - MySQL management tool . Al aplicar los pasos que pide al ejecutar Help: Start extension bisect, dice que el error se debe a la extension formulahendry.vscode-mysql . Por favor, decirme cómo puedo solucionar este problema. 

Extension version: 0.4.1
VS Code version: Code 1.83.0 (e7e037083ff4455cf320e344325dacb480062c3c, 2023-10-03T16:12:16.321Z)
OS version: Windows_NT x64 10.0.19045
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
![extension formulahendry vscode-mysql](https://github.com/formulahendry/vscode-mysql/assets/135270111/10f7f6b8-6d3e-4844-84ae-9441773ce1e5)

![Error Cannot read properties of null (reading 'getPassword')](https://github.com/formulahendry/vscode-mysql/assets/135270111/f4a63a2f-5640-4f80-a7d0-757886c1f798)

|CPUs|Intel(R) Core(TM) i3-2310M CPU @ 2.10GHz (4 x 2095)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: unavailable_off<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: unavailable_off<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|5.91GB (2.31GB free)|
|Process Argv|--crash-reporter-id e91d5f2a-5db3-43fd-b9bc-171c35c9dd32 --crash-reporter-id e91d5f2a-5db3-43fd-b9bc-171c35c9dd32|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256860
vstes627:30244334
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30294714
vstes263:30335439
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30410667
py29gd2263:30856252
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
2e4cg342:30602488
f6dab269:30613381
a9j8j154:30646983
showlangstatbar:30737416
962ge761:30841074
03d35959:30757346
pythonfmttext:30731395
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
pythonnosmt12:30797651
pythonidxptcf:30805731
pythonnoceb:30805159
copilotsettingc:30839828
synctok:30821570
dsvsc013:30795093
dsvsc014:30804076
diffeditorv2:30821572
dsvsc015:30845448

```

</details>

![Error Cannot read properties of null (reading 'getPassword')](https://github.com/formulahendry/vscode-mysql/assets/135270111/5d579a44-93d6-427c-b0d0-20002b877784)
![extension formulahendry vscode-mysql](https://github.com/formulahendry/vscode-mysql/assets/135270111/da5738dc-300f-4158-bda8-68d9c93a13dd)
#Cannot read properties of null (reading 'getPassword')

**Comment by LauSere** [comment]
@gjsjohnmurray

**Comment by formulahendry** [comment]
Fixed in v0.5.0
